indigo_app/views/works.py

```diff
@@ -559,6 +560,16 @@ def get_works(self, table):
                         info['status'] = 'success'
                         info['work'] = work
 
+                        # TODO: neaten this up
+                        if not row.get('commenced_by') == u"":
```

Comment:
how about `if row.get('commenced_by'):` because non-empty strings are thruthy, empty strings are falsey.

Reply:
of course, thanks – I initially just had it as `info['commenced_by'] = row.get('commenced_by')` and relied on this to be `None` or similar lower down, but that didn't work, so I went overboard in the other direction